### PR TITLE
Added iOS vision cocoapods podspec template

### PR DESCRIPTION
**Pull request by priankakariatyml**
1. Added pod spec template for vision library.
2. Updated shell script to build any iOS framework based on a variable.
3. Added Umbrella header and module map for vision

### CODE REVIEW

File: tensorflow_lite_support/tools/ci_build/builds/build_ios_framework.sh [hunk 1]
```diff
@@ -70,23 +86,74 @@ function create_framework_archive {
...
+  #  |   |-- c/
   #  |   |   +-- <C-API header files...>
   #  |   +-- ios/
   #  |       +-- <Obj-C header/source files...>
   #  +-- Frameworks/
   #      +-- TensorFlowLiteTaskTextC.framework
   #
 
-  # ----- (1) Copy source files -----
+  # ----- (1) Copy ssource files -----
```
Comment (yyoon) [inline tensorflow_lite_support/tools/ci_build/builds/build_ios_framework.sh:97]: Typo?

File: tensorflow_lite_support/tools/ci_build/builds/build_ios_framework.sh [hunk 2]
```diff
@@ -70,23 +86,74 @@ function create_framework_archive {
...
+        */ios/task/text/*/Sources/*
+        */ios/task/text/apis/*
+      "
+
+      IOS_HEADER_PATTERNS="
+        */ios/task/text/*/Sources/*.h
+      "
+    ;;
+    *)
+      echo "Framework Name Provided is not in the list of buildable frameworks."
```
Comment (yyoon) [inline tensorflow_lite_support/tools/ci_build/builds/build_ios_framework.sh:153]: Maybe also print out what values are currently accepted?

File: tensorflow_lite_support/tools/ci_build/builds/build_ios_framework.sh [hunk 3]
```diff
@@ -16,6 +16,7 @@
 
 # Set the following variables as appropriate.
 #   * BAZEL: path to bazel. defaults to the first one available in PATH
+#   * FRAMEWORK_NAME: Name of the iOS framework to be built.
```
Comment (yyoon) [inline tensorflow_lite_support/tools/ci_build/builds/build_ios_framework.sh:19]: Please specify which names are currently accepted.

File: tensorflow_lite_support/tools/ci_build/builds/build_ios_framework.sh (hunk 1 above)
Comment (priankakariatyml) [reply tensorflow_lite_support/tools/ci_build/builds/build_ios_framework.sh]: Fixed

File: tensorflow_lite_support/tools/ci_build/builds/build_ios_framework.sh (hunk 3 above)
Comment (priankakariatyml) [reply tensorflow_lite_support/tools/ci_build/builds/build_ios_framework.sh]: Added

File: tensorflow_lite_support/tools/ci_build/builds/build_ios_framework.sh (hunk 2 above)
Comment (priankakariatyml) [reply tensorflow_lite_support/tools/ci_build/builds/build_ios_framework.sh]: Done

**Review by yyoon — APPROVED** [review]

File: tensorflow_lite_support/ios/TensorFlowLiteTaskVision.podspec.template [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  s.name             = 'TensorFlowLiteTaskVision'
+  s.version          = '${TFLS_BUILD_VERSION}'
+  s.authors          = 'Google Inc.'
+  s.license          = { :type => 'Apache',:file => "LICENSE.txt"}
+  s.homepage         = 'https://github.com/tensorflow/tflite-support'
+  s.source           = { :http => '${TFLS_DOWNLOAD_URL}' }
+  s.summary          = 'TensorFlow Lite Task Library - Vision'
+  s.description      = 'The Natural Language APIs of the TFLite Task Library'
+
+  s.ios.deployment_target = '13.0'
```
Comment (khanhlvg) [inline tensorflow_lite_support/ios/TensorFlowLiteTaskVision.podspec.template:11]: @priankakariatyml Can this be reduced to 9.0 as in the Text Library and TFLite? 

iOS 13 was released in 2019, which is too recent. We want to support older devices as well.

Comment (khanhlvg) [inline tensorflow_lite_support/ios/TensorFlowLiteTaskVision.podspec.template:11]: @priankakariatyml Can this be reduced to 9.0 as in the Text Library and TFLite? 

iOS 13 was released in 2019, which is too recent. We want to support older devices as well.

Comment (khanhlvg) [reply tensorflow_lite_support/ios/TensorFlowLiteTaskVision.podspec.template:11]: We want to support at least to iOS 10, which is the current supported version of ML Kit. Both depends on GMLImage.
cc: @lu-wang-g

Comment (priankakariatyml) [reply tensorflow_lite_support/ios/TensorFlowLiteTaskVision.podspec.template]: I have reduced it to 10.0. 9.0 is too old. Some of the Core Image apis are not available in 9.0.

**Comment by khanhlvg** [comment]
Merged.

**Pull request closed without merging**